.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,33 +1,39 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: "[bug]"
+title: '[bug]'
 labels: bug
 assignees: ''
-
 ---
 
 ### Describe the bug
+
 A clear and concise description of what the bug is.
 
 ### To Reproduce
+
 Steps to reproduce the behavior:
+
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
 ### Expected behavior
+
 A clear and concise description of what you expected to happen.
 
 ### Screenshots
+
 If applicable, add screenshots to help explain your problem.
 
 ### Environment (please complete the following information):
- - _OS_: [e.g. iOS, Windows, MacOS]
- - _Browser_: [e.g. chrome, safari, iOS/android app]
- - _Version_: [e.g. 22]
- - _Device_: [e.g. iPhone6]
+
+- _OS or Device_: [e.g. iOS, Windows, MacOS, iPhone 6]
```

Comment:
Might be a bit redundant to have `iOS` if iPhone 6 is there. I guess for phones it's less important of OS as much as app version is.

```suggestion
- _OS or Device_: [e.g. Windows, MacOS, iPhone 6]
```